nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -444,6 +446,12 @@ def model_provider_func(self, pre_process, post_process):
             )
             if self.cfg.get("apply_embedding_scaling", False) and parallel_state.is_pipeline_first_stage():
```

Comment:
can we move these extend instance (for gemma1 and 2) to a method maybe called "def model_customize(self)"? and here only do self.model_customize. Please use your ideas if you have better naming.

Reply:
sure that's a good idea